x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_flyout.tsx

```diff
@@ -84,35 +84,9 @@ export const CreateTrustedAppFlyout = memo<CreateTrustedAppFlyoutProps>(
       };
     }, [isLoadingPolicies, policyList]);
 
-    const creationErrorsMessage = useMemo<string | undefined>(() => {
-      let errorMessage = creationErrors
-        ? CREATE_TRUSTED_APP_ERROR[creationErrors.message.replace(/(\[(.*)\]\: )/, '')] ||
-          creationErrors.message
-        : undefined;
-
-      if (
-        creationErrors &&
-        creationErrors.attributes &&
-        creationErrors.attributes.type === 'TrustedApps/PolicyNotFound'
-      ) {
-        policies.options.forEach((policy) => {
-          errorMessage = errorMessage?.replace(policy.id, policy.name);
-        });
-      } else if (
-        creationErrors &&
-        creationErrors.attributes &&
-        creationErrors.attributes.type === 'EndpointLicenseError'
-      ) {
-        errorMessage = i18n.translate(
-          'xpack.securitySolution.trustedapps.createTrustedAppFlyout.byPolicyLicenseError',
-          {
-            defaultMessage:
-              'Your Kibana license has been downgraded. As such, individual policy configuration is no longer supported.',
-          }
-        );
-      }
-      return errorMessage;
-    }, [creationErrors, policies]);
+    const creationErrorsMessage = useMemo<CreateTrustedAppFormProps['error'] | undefined>(() => {
+      return creationErrors?.message ?? [];
```

Comment:
We no longer need to do API error parsing. The errors returned from the Lists plugin are much more legible than the prior ones we got from the Kibana platform's `schema` module